old/index.php

```diff
@@ -162,7 +162,6 @@
       <div class="wrapper">
         <ul>
           <li><img src="/images/thx_paper.png" alt="Paper" title="Paper" /></li>
-          <li><img src="/images/thx_tuinity.png" alt="Tuinity" title="Tuinity" /></li>

```

Comment:
delete the `thx_tuinity.png` image as well

Reply:
Done.